cloudbuild.yaml

```diff
@@ -0,0 +1,357 @@
+
+- id: Get Kubernetes Credentials
+  name: gcr.io/cloud-builders/gcloud
+  args:
+  - container
+  - clusters
+  - get-credentials
+  - "limani-integ"
+  - --region
+  - "us-central1"
+  - --project
+  - "$PROJECT_ID"
+
+- id: Initialized
+  name: bash
+  waitFor:
+  - Initialize git
+  - Get Kubernetes Credentials
+
+# Airflow Operator Verification
+
+- id: Build Airflow Operator
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  dir: k8s/airflow-operator
+  args:
+  - -exc
+  - make -j4 app/build
+
+- id: Verify Airflow Operator
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  - Build Airflow Operator
+  env:
+  - APP_DEPLOYER=gcr.io/cloud-marketplace-ops-test/airflow-operator/deployer:alpha
+  - |
+    APP_PARAMETERS={
+                     "testerImage": "gcr.io/cloud-marketplace-ops-test/airflow-operator/tester:alpha"
+                   }
+  args:
+  - -exc
+  - |
+    /scripts/driver_internal.sh \
+        --deployer="$$APP_DEPLOYER" \
+        --parameters="$$APP_PARAMETERS"
+
+# Cassandra Verification
+
+- id: Build Cassandra
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  dir: k8s/cassandra
+  args:
+  - -exc
+  - make -j4 app/build
+
+- id: Verify Cassandra
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  - Build Cassandra
+  env:
+  - APP_DEPLOYER=gcr.io/cloud-marketplace-ops-test/cassandra/deployer:3.11
+  - APP_PARAMETERS={}
+  args:
+  - -exc
+  - |
+    /scripts/driver_internal.sh \
+        --deployer="$$APP_DEPLOYER" \
+        --parameters="$$APP_PARAMETERS"
+
+# Elastic GKE Logging Verification
+
+- id: Build Elastic GKE Logging
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  dir: k8s/elastic-gke-logging
+  args:
+  - -exc
+  - make -j4 app/build
+
+- id: Verify Elastic GKE Logging
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  - Build Elastic GKE Logging
+  env:
+  - APP_DEPLOYER=gcr.io/cloud-marketplace-ops-test/elastic-gke-logging/deployer:6.3
+  - APP_PARAMETERS={}
+  args:
+  - -exc
+  - |
+    /scripts/driver_internal.sh \
+        --deployer="$$APP_DEPLOYER" \
+        --parameters="$$APP_PARAMETERS"
+
+# Elasticsearch Verification
+
+- id: Build Elasticsearch
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  dir: k8s/elasticsearch
+  args:
+  - -exc
+  - make -j4 app/build
+
+- id: Verify Elasticsearch
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
+  waitFor:
+  - Initialized
+  - Build Elasticsearch
+  env:
+  - APP_DEPLOYER=gcr.io/cloud-marketplace-ops-test/elasticsearch/deployer:6.3
+  - APP_PARAMETERS={}
+  args:
+  - -exc
+  - |
+    /scripts/driver_internal.sh \
+        --deployer="$$APP_DEPLOYER" \
+        --parameters="$$APP_PARAMETERS"
+
+# Grafna Verification
+
+- id: Build Grafna
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
```

Comment:
We don't want to always use `latest`--the tag should match the verions of the vendored tools.

I filed #204 to track this. Please address it as part of your ongoing work with the `dev` image.

Reply:
I'll hold off on merging this until https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/200 is merged, and I'll update and ping to a tag.